jvb/src/test/kotlin/org/jitsi/videobridge/SpeechActivityTest.kt

```diff
@@ -77,17 +79,17 @@ class SpeechActivityTest : ShouldSpec() {
 
             conferenceSpeechActivity.orderedEndpoints shouldContainExactly listOf(a, b, c, d)
 
-            every { a.isSendingVideo } returns false
+            every { a.videoType } returns NONE
```

Comment:
`isSendingVideo` still exists, right?  It seems like `SpeechActivityTest` would only care about it sending/not sending rather than the specific video type (or maybe it _does_ care for a reason I'm missing?).

Reply:
It cares about whether video is available or not, which is equivalent to `videoType == NONE`. I can expose a boolean from Endpoint, but I'm planning to use the exposed `videoType` elsewhere too (bandwidth allocation), so it wouldn't be a huge win.

Reply:
Yeah, my thought was that code that only cares about whether or not the endpoint is sending anything (which this test appears to) should use `isSendingVideo`.  Code that cares about what _kind_ of video the endpoint is sending would use `videoType`.